stl/inc/vector

```diff
@@ -3959,6 +3959,28 @@ _EXPORT_STD struct sorted_equivalent_t {
     explicit sorted_equivalent_t() = default;

```

Comment:
Pre-existing: we could use a comment on this `_HAS_CXX23` section clarifying why all of this `flat_meow` support machinery is here.